lib/vmdb/loggers.rb

```diff
@@ -40,7 +40,16 @@ def self.apply_config(config)
     def self.create_loggers
       path_dir = Rails.root.join("log")
 
-      $log               = VMDBLogger.new(path_dir.join("evm.log"))
+      $evm_log = VMDBLogger.new(path_dir.join("evm.log"))
+
+      $log =
+        if ENV["CONTAINER"]
+          stdout_log = StdoutLogger.new.tap { |l| l.formatter = ContainerFormatter.new }
+          MulticastLogger.new($evm_log, stdout_log)
```

Comment:
This is more question than request for change:  Is the logging output that is more visible to the user when in a container environment the `evm.log` file or STDOUT (this is without a common logging interface to view the logs)?

The reason I ask is because in my humble opinion, viewing json formatted logs directly is not user readable, and requires something like kibana to view and digest them sensibly.  Obviously json formatted logs makes it machine readable without introducing a parser, but makes the human parseability much less so.

So my follow up to that is do we see users, when the common logging stack is unavailable, using the stdout log or the evm.log to monitor the logs within a container/pod?  My request would be to format the more user facing one (which might be what you already have here) with the traditional log formatter, and the one to be parsed by fluentd/logstash be formatted in json.